packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * WordPress dependencies
+ */
+const { camelCaseDash } = require( '@wordpress/scripts/utils' );
```

Comment:
This will introduce a cyclic dependency between `@wordpess/scripts` and this package.

I think it's fine to move it to this package to resolve it.

It's used here as well:
https://github.com/WordPress/gutenberg/blob/master/webpack.config.js#L17

However, it doesn't matter where it is going to be imported from.

Reply:
I'm becoming quite worried about the spread of this utility, given the repeated cautions on the fact that it doesn't produce accurate results.

https://github.com/WordPress/gutenberg/pull/13814/files#r258035009

Reply:
> I think it's fine to move it to this package to resolve it.

This webpack plugin would seem like a strange place to provide this util.

I _would_ like to remove the dependence on `@wordpress/scripts` from this package just to use `camelCaseDash`. It's trivial functionality, but `@wordpress/scripts` pulls in a lot of heavy packages like pupeteer.

Do folks have a clear idea of the best path forward?

Reply:
Can we copy it for now? I will tackle it in the follow-up issue. I'm actually opening a new one based on feedback from @aduth where we can also discuss how to solve the issue with `escape-html` -> `escapeHTML` vs `escapeHtml` (which I personally don't quite know how to automate).

Reply:
+1 to copy for now

Reply:
I've copied the function to this package for now.